app/vmbackup/README.md

```diff
@@ -9,9 +9,9 @@ Supported storage systems for backups:
 * Any S3-compatible storage such as [MinIO](https://github.com/minio/minio), [Ceph](https://docs.ceph.com/docs/mimic/radosgw/s3/) or [Swift](https://www.swiftstack.com/docs/admin/middleware/s3_middleware.html). See `-customS3Endpoint` command-line flag.
 * Local filesystem. Example: `fs://</absolute/path/to/backup>`
 
-Incremental backups and full backups are supported. Incremental backups are created automatically if the destination path already contains data from the previous backup.
+Incremental backups and full backups supported. Incremental backups created automatically if the destination path already contains data from the previous backup.
```

Comment:
It was correct in both cases before. But if you don't like how it sounds - feel free to re-phrase. Like "vmbackup supports incremental and full backups."